config/capacities.json

```diff
@@ -323,5 +323,21 @@
             "wind": 3,
             "solar": 532
         }
+    },
+    "US": {    
+        "capacity": {
+            "_comment": "distributed PV: 9778MW, not included below",
```

Comment:
Still needs to go one level up in hierarchy (the order of keys does not matter in a dictionary, so the change you made did not change anything).
Basically you need 
```json
"US: {
  "capacity": {
    ...
  },
  "_comment": "...."
}
```